sources/ci/yml-structure.md

```diff
@@ -99,6 +102,7 @@ A brief overview of each section of the yml is provided in this table. For a det
 |&nbsp;&nbsp;&nbsp;&nbsp;[post_ci:](build-and-test/)        | no default                                                                                           | Include commands that are not really a part of your core CI workflow but should be run after CI finishes. Commands in this section are run inside your CI container. [Read more](build-and-test/)                                                                                                                                                                                                  |
 |&nbsp;&nbsp;&nbsp;&nbsp;[on_success:](build-and-test/)      | no default                                                                                           | Include commands you want to execute only if your CI workflow passes, i.e. the ci section exits with 0.  Commands in this section are run inside your CI container. [Read more](build-and-test/)                                                                                                                                                                                                  |
 |&nbsp;&nbsp;&nbsp;&nbsp;[on_failure:](build-and-test/)      | no default                                                                                           | Include commands you want to execute only if your CI workflow fails, i.e. the ci section does not exit with 0.  Commands in this section are run inside your CI container. [Read more](build-and-test/)                                                                                                                                                                                           |
+|&nbsp;&nbsp;&nbsp;&nbsp;[always:](build-and-test/)      | no default                                                                                           | Include commands you want to execute irrespective of CI workflow passes or fails..  Commands in this section are run inside your CI container. [Read more](build-and-test/)                                                                                                                                                                                           |
```

Comment:
Include commands you want to execute irrespective of CI workflow status.  Commands in this section are run inside your CI container. [Read more](build-and-test/)